Persistence/Repositories/MemberRepository.cs

```diff
@@ -45,3 +44,13 @@ public async Task Save(CancellationToken cancellationToken)
     }
 
 }
+
+public interface IMemberRepository
```

Comment:
This interface should remain in the Domain project. See https://stackoverflow.com/a/72399927 for reasons why